runner/ansible/callback_plugins/trento.py

```diff
@@ -279,6 +280,13 @@ def _get_playbook_id(self):
         playbook_id = play["results"][0]["playbook"]
         return playbook_id
 
+    def _return_timestamp(self):
+        """
+        Return a timestamp
+        """
+        timestap = datetime.now().strftime("%H:%M:%S")
+        return timestamp
+
```

Comment:
Aren't we suppose to use this function somewhere? Or it's just a conventional callback?

Reply:
Just an entrypoint, babysteps

Reply:
Fair enough, I thought this was ready for the review, sorry :smile: if you want you can mark PRs as draft to make explicit that it's still an ongoing effort :+1: